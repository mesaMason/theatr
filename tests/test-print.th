func main() -> int:

    int x
    
    print(5)
    print("hello world!")
    print(true)

    print(4+3)
    
    x = 10
    print(x)

    return 0
