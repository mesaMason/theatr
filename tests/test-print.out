5
hello world!
true
7
10
